admin_guide/syncing_groups_with_ldap.adoc

```diff
@@ -0,0 +1,468 @@
+= Syncing Groups With LDAP
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+As an OpenShift administrator, you may want to group users in order to better manage them, place
+appropriate permissions or to enhance collaboration. Your organization may already have created such
+groups and stored them in an LDAP server. OpenShift allows for you to sync those records with
+internal OpenShift records so that you can focus on managing your groups in once place only.
+OpenShift currently supports group sync with LDAP servers using three common schema for defining
+group membership: RFC2307, Active Directory, and augmented Active Directory.
+
+[IMPORTANT]
+====
+These solutions are a work in progress. As improvements are made,
+these instructions will be simplified.
+====
```

Comment:
> This was stuck in there before dev-cut. @deads2k Do we expect major changes anymore?

Just make sure they are accurate.  We may change how a few things are spelled, but no concepts are changing.